src/tester.rs

```diff
@@ -452,12 +452,16 @@ fn run_tests(
                 let mut pass_stderr = None;
                 let mut pass_stdout = None;
 
-                let maybe_test = tests.get(&cmd_name);
-
-                if let Some(t) = maybe_test {
-                    cmd.args(&t.args);
+                let default_test = TestCmd {
+                    status: Some(Status::Success),
+                    stderr: None,
+                    stdout: None,
+                    args: Vec::new(),
                 };
 
+                let test = tests.get(&cmd_name).unwrap_or(&default_test);
```

Comment:
I agree, but unfortunately `tests.get()` returns a reference from the HashMap so we need build the `TestCmd` struct ahead of time and return a reference to it in the closure.

Reply:
OK. Maybe, then, a function `default_test` would be a good halfway house? It will make things a bit clearer, and reduce the size of the monster that is `run_tests`.

Reply:
What about something like this? We can't call the function inside the argument to `unwrap_or` as it will create a temporary whose lifetime is shorter than that of `run_tests`.

Reply:
It's not quite what I hoped, but I agree with your reasons re: the borrow checker. Time to squash!